compile/README.md

```diff
@@ -26,11 +38,34 @@ NOTE: The files are not statically compiled, and hence any pypi dependencies wil
     ```
 2. **Variables**
     - `REMOVE_BRANCHES: "true"` removes all destination branches except main before pushing
-    - `GIT_STRATEGY: clone` !Important as it makes sure that source repo doesn't contain cached files
-    - `GIT_DEPTH: 1` can be set accordingly
+    - (Required) `GIT_STRATEGY: clone` !Important as it makes sure that source repo doesn't contain cached files
+    - `GIT_DEPTH: 1` can be set accordingly. By default it is set to 20 by gitlab.
     - `GIT_SUBMODULE_DEPTH: 1` can be set accordingly
     - `FILES_TO_COPY`: Use this as gitlab multiline variable and provide full path of files/folders that needs to be copied to destination. No `.py` or `.so` files will be copied using this variable. It will skip such files, even if you give entire folder to copy.
-    - `DELETE_BEFORE_COMPILING`: Use this as gitlab multiline variable and provide full path for `.py` files that needs to be deleted before cythonization
+    - (Optional) `DELETE_BEFORE_COMPILING`: Use this as gitlab multiline variable and provide full path for `.py` files that needs to be deleted before cythonization
+
+    Below is the configuration for `Video Relay - Smart` repo variables for reference
+
+    ```yaml
+    variables:
+        # pycompile variables
+        FILES_TO_COPY: |
+            ./Dockerfile
+            ./docker-compose.yml
+            ./docker-compose.production.yml
+            ./USAGE.md
+            ./TEST.md
+            ./README.md
+            ./refresh_setup.sh
+            ./requirements.txt
+            ./misc/config.toml
+            ./CHANGELOG.md
+            ./refresh_setup.sh
+            ./test/README.md
+        GIT_STRATEGY: clone
+        GIT_DEPTH: 1
+        REMOVE_BRANCHES: "true"
```

Comment:
You needn't and shouldn't reference any specific application or internal repository that is using this pipeline. You can try to give generic examples.